client/src/pages/about/AboutContact.tsx

```diff
@@ -13,7 +14,10 @@ interface AboutContactProps {
 const UnstyledAboutContact: FunctionComponent<AboutContactProps> = ({
   className,
 }) => {
-  return <Box>Contact page</Box>;
+  return (<>
+    <Contact />
+  </>
+    );
 };
 
-export const AboutContact = withTheme(styled(UnstyledAboutContact)``);
+export const AboutContact = withTheme(styled(UnstyledAboutContact)``);
```

Comment:
Is this `AboutContact` getting used anywhere?  looks like it's just exporting `Contact` component